active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,356 @@
+and why to use this API. -->
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+The absence of a standard UI pager control, last seen as DataPager in Silverlight, has been a pain point in both WPF and UWP that has forced a variety of unfavorable workarounds for developers. Inclusion of this control in UWP would resolve an ecosystem gap and enable developers to efficiently deliver UI paging experiences in their applications. Through XAML Islands, it may also provide an opportunity to benefit WPF developers working with DataGrid or ListView. The scope of this proposal is not to provide a data paging solution at this time, but to start with UI paging as a foundation on top of which data paging may be later integrated.
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Pager control is a customizable pager UI widget with generic events that can be configured to navigate pages for several view controls.
+
+**Important APIs:** [PagerControl class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.pagercontrol)
+
+Pager control can be configured to use a ComboBox, NumberBox, or numeric Button panel as the core pager control.
+
+## Is this the right control? 
+
+Use a **PagerControl** to create an accessible navigation interface for views that have multiple pages to display. 
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+A pager control can have several configurations, including these notable ones. 
+
+### ComboBox Pager Control
+
+![](images/pager-control-combobox.png)
+
+XAML
+```XAML
+<GridView x:Name="gridView1" ... />
+<controls:PagerControl x:Name="MainPagerControl"
+        DisplayMode="ComboBox"
+	NumberOfPages="10"
```

Comment:
How does an app know how to set this? It determines on the amount of content and the control's actual size, doesn't it?

Reply:
Are you referring to the NumberOfPages property?

Reply:
Right. As an app, I don't know how many pages there should be; it depends on the size of the list etc. I could see this being a read-only property, but I can't think of a case where an app could or should set it.

Reply:
It doesn't necessarily depend on the size of the list. For example, you could always present 10 items per page regardless of the available viewport. See [my comment here](https://github.com/microsoft/microsoft-ui-xaml-specs/pull/88#discussion_r451089115) for more context.

Reply:
This is a property that the developer will set so the pager control will not how many buttons/indices to navigate through. Since this is just a UI component, the pager control will not know anything about the data source or layout view in order to determine how many pages there should be.

Reply:
I think the way it is currently, the app would have to set this based on number of items, the layout it is using and the size of each item since the list and pager are decoupled.

Reply:
Yes, @Ranjesh is correct that is how it currently works. The developer would need to determine how many pages there should be based on the layout and content being displayed so it would need to be set as well.